packages/angular-cli/models/webpack-configs/production.ts

```diff
@@ -13,7 +13,6 @@ export const getProdConfig = function (wco: WebpackConfigOptions) {
       new webpack.DefinePlugin({
         'process.env.NODE_ENV': JSON.stringify('production')
       }),
-      new webpack.LoaderOptionsPlugin({ minimize: true }),
```

Comment:
I *think* in general this is still needed, because it signals all loaders to minimize. 

So we do it for JS via `uglifyjs`, and for CSS via `cssnano`, but I assume (haven't tested) that the some of the other loaders (file/url loader perhaps) are also affected?

@thelarkin do you know?